guides/common/modules/proc_using-a-third-party-repository.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Using_a_Third_Party_Repository_{context}"]
+= Using a Third Party Repository
+
+In this scenario, you use a third party repository to sync content.
+
+.On a connected server
+. In the connected server of the {ProjectWebUI}, navigate to *Content* > *Product*.
+. Click on the third party product.
+. Click over the third party repository.
+On this page, you can see the link in front of *Published At:*.
+Copy the information in the link.
+
+.On a disconnected server
+You need to create your third party product and third party repository.
+Create your third party product with the following:
+
+. Upstream URL: This is the same information copied from the steps in the connected server.
```

Comment:
I would probably just repeat the procedure to copy it. This also goes with what I said in this comment.